kagiso_smart_404/utils.py

```diff
@@ -1,17 +1,29 @@
+import re
+
 from wagtail.wagtailcore.models import Page
 
 
+def get_instant_redirect(slug, root_page):
+    cleaned_slug = re.match('/.*/(.*)/', slug).group(1)
```

Comment:
this needs a comment to explain what you are doing and how the regex works, right now it is a dark art
, eg. # input =/my/url/, slug=url